tests/ethereum/ethereum.spec.ts

```diff
@@ -0,0 +1,642 @@
+import { parseEther } from "ethers/lib/utils";
+import {
+  MAX_UINT_AMOUNT,
+  oneEther,
+  WRAPPED_NATIVE_TOKEN_PER_NETWORK,
+} from "./../../helpers/constants";
+import { waitForTx } from "./../../helpers/utilities/tx";
+import { impersonateAddress } from "./../../helpers/utilities/fork";
+import { AaveEcosystemReserveController__factory } from "./../../typechain/factories/@aave/periphery-v3/contracts/treasury/AaveEcosystemReserveController__factory";
+import { Ownable } from "./../../dist/types/typechain/@aave/core-v3/contracts/dependencies/openzeppelin/contracts/Ownable.d";
+import { getTreasuryAddress } from "./../../helpers/market-config-helpers";
+import { getERC20, getFlashLoanLogic } from "./../../helpers/contract-getters";
+import {
+  getEthersSigners,
+  getFirstSigner,
+} from "./../../helpers/utilities/signer";
+import { AaveEcosystemReserveV2__factory } from "./../../typechain/factories/@aave/periphery-v3/contracts/treasury/AaveEcosystemReserveV2__factory";
+import {
+  INCENTIVES_PROXY_ID,
+  TREASURY_CONTROLLER_ID,
+  TREASURY_IMPL_ID,
+  TREASURY_PROXY_ID,
+} from "./../../helpers/deploy-ids";
+import { expect } from "chai";
+import { BigNumber } from "ethers";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import {
+  aave,
+  ATOKEN_IMPL_ID,
+  ConfigNames,
+  DELEGATION_AWARE_ATOKEN_IMPL_ID,
+  ETHEREUM_SHORT_EXECUTOR,
+  getAaveOracle,
+  getAaveProtocolDataProvider,
+  getACLManager,
+  getAToken,
+  getEthersSignersAddresses,
+  getIncentivesV2,
+  getIRStrategy,
+  getPool,
+  getPoolAddressesProvider,
+  getPoolAddressesProviderRegistry,
+  getPoolConfiguratorProxy,
+  getStableDebtToken,
+  getUiIncentiveDataProvider,
+  getUiPoolDataProvider,
+  getVariableDebtToken,
+  getWrappedTokenGateway,
+  ICommonConfiguration,
+  INCENTIVES_V2_IMPL_ID,
+  loadPoolConfig,
+  PoolAddressesProvider,
+  POOL_CONFIGURATOR_IMPL_ID,
+  POOL_IMPL_ID,
+  STABLE_DEBT_TOKEN_IMPL_ID,
+  strategyWETH,
+  VARIABLE_DEBT_TOKEN_IMPL_ID,
+  ZERO_ADDRESS,
+} from "../../helpers";
+
+// Prevent error HH9 when importing this file inside tasks or helpers at Hardhat config load
+declare var hre: HardhatRuntimeEnvironment;
+
+const InitializeError = "Contract instance has already been initialized";
+
+describe("Ethereum V3 - Token-less deployment", function () {
```

Comment:
Should we break this file up into a few files its > 600 lines

Reply:
agree! splitted it into governance, implementations, pool configs, roles and periphery tests.